app/models/pipeline/logic_review.py

```diff
@@ -92,20 +92,11 @@ async def review(self, mr: MergeRequest) -> PipelineResult:
                 summary="解析审查响应失败", comments=[]
```

Comment:
[From AI Code Reviewer][bot] The per-file comment limit of 2 has been removed.  Was this intentional?  If so, update relevant documentation.